src/Adapter/CartRule/CommandHandler/AddCartRuleHandler.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
     public function handle(AddCartRuleCommand $command): CartRuleId
     {
         //@todo: restrictions are missing. We should consider dedicated command for restrictions
-        $cartRule = $this->cartRuleRepository->create($this->buildCartRuleFromCommandData($command));
+        $cartRule = $this->cartRuleRepository->add($this->buildCartRuleFromCommandData($command));
```

Comment:
optional change. I switched to "add" because I think it makes sense to have add - when providing already filled object model, create -when required properties for creation are passed and the model is built inside

Reply:
Indeed!